app/routes/reductions.js

```diff
@@ -0,0 +1,76 @@
+const organisationUnitConstants = require('../constants/organisation-unit')
+const reductionsService = require('../services/reductions-service')
+const Reduction = require('../services/domain/new-reduction')
+
+module.exports = function (router) {
+  router.get('/:organisationLevel/:id/reductions', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+    var id = req.params.id
+    var successText = req.body.successText ? req.params.successText : null
```

Comment:
Should this be `req.body.successText` in both places?

Reply:
Yea, so it checks if it has a value, if it does, we use it, else null.

Reply:
Yeh, but here is using `req.body.successText` in the check and `req.params.successText` and the output. Should they both be `req.body.successText`?

Reply:
Good spot, should be fixed now.